### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by jrmlhermitte**
A little messy, but these should test the data formats.

Also corrected some small typos in `handlers.py`

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
tests passed, this is ready for review

File: suitcase/pizza_box/handlers.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def __init__(self, resource_path, chunk_size=1024):
             chunk = chunk[column_keys]
             self.chunks_of_data.append(chunk)
 
-    def __call__(self, chunk_num, column):
+    def __call__(self, chunk_num, column=0):
```
Comment (danielballan) [inline suitcase/pizza_box/handlers.py]: Is this default value necessary? The only reason I can think of that a Handler would need a default value is if there are old Datum documents missing a parameter that was later added.

File: suitcase/pizza_box/tests/test_handlers.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+def convert_time(ts, tns):
+    return ts + tns*1e-9
+
+
+# for the encoder pizza box
+def enc2counts(x):
+    return int(x) if int(x) <= 0 else -(int(x) ^ 0xffffff - 1)
+
+
+# tmpdir is a pytest fixture
```
Comment (danielballan) [inline suitcase/pizza_box/tests/test_handlers.py]: Neat. Did not know about this.

File: suitcase/pizza_box/handlers.py (hunk 1 above)
Comment (jrmlhermitte) [reply suitcase/pizza_box/handlers.py]: Yes it is unfortunately. The reason is to prove backwards compatibility with the older version of the handler which doesn't know about this. Currently ISS doesn't have a `column` kwarg but QAS does.
The other option is to keep two different handlers. If we do go this route, we should do the same for eigerhabdler which is also following this pattern (eiger handler has two keys of which either is used depending on the version)

Comment (jrmlhermitte) [reply suitcase/pizza_box/handlers.py]: (sorry for typos phone autocorrect)

**Pull request closed without merging**